feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/README.md

```diff
@@ -0,0 +1,38 @@
+# RT-1.21: BGP TCP MSS and PMTUD 
+
+## Summary
+
+BGP TCP MSS and PMTUD
+
+## Procedure
+
+*   Establish BGP sessions between:
+    *   ATE port-1 ---      eBGP-IPv4/IPv6        ---- DUT1 
+    *   ATE port-1 ----    Multihop iBGP IPv4     ---- DUT2.
+*   DUT-2 is directly connected to DUT-1.  
+*   TODO : Verify that the default TCP MSS value is set below interface MTU value.
```

Comment:
Why these TODOs?

Reply:
Removed TODO and added deviation. please check